deployments/cloudfoundry/tile/scripts/README.md

```diff
@@ -1,38 +1,77 @@
+The next few steps will be run from the root directory of the Splunk OpenTelemetry Collector
 
-**Note:** This script must be run from the top-level directory of this repository to properly account for the
-`license_finder` tool's contents.
+- You've already installed the license_finder tool at this point. Add the known licenses using the tool. Copy paste the contents of add_known_licenses.txt to your terminal and hit enter
+- Create an OSDF (open source disclosure file).
+  - Use license_finder to determine if there are any new dependencies by running the following command:
 
-**Sample usage:**
 ```shell
-# When creating the tile locally the version can be found in deployments/cloudfoundry/tile/tile-history.yml
-$ python3 deployments/cloudfoundry/tile/scripts/generate_osdf.py --otelcol_version 0.X.0
+$ license_finder # Will show if any dependencies need to be approved
+Dependencies that need approval:
 ```
 
-## install_cli_dependencies.sh
+- If you don’t see the Dependencies that need approval message
+  - Use generate_osdf.py script to generate OSDF file. Output will be in the local file “OSDF VX.XX.X.txt”
+  ```shell
+    python deployments/cloudfoundry/tile/scripts/generate_osdf.py --otelcol_version <version from tile-history.yml>
+  ```
+- Otherwise, you will need to manually approve each dependency. This will likely follow the format of the original license_finder commands, but you’ll have to determine which license applies to a given dependency, or if a new license can be approved or not.
+  - For guidance on whether a license is approved for use, reach out to Aunsh, legal, or the Broadcom team through the Pivotal Partners Slack using the private #signalfx channel.
+  - Any commands that needed to be run should be added to the list of commands above, so future users don’t have to duplicate your work.
 
-This is a shell script used to install all of the CLI tools that are required for development
-working on the Tanzu Tile. This script was made for MacOS, and has not been
-tested on any other OS at this time.
+- Next step is to make the actual release by uploading the artifiacts and filling out a form. The process is outlined [here](https://drive.google.com/file/d/1lIJly4qS4drsE0jhmk6ZcgA3AS80hzWP/view) under `Publishing new releases of your product`
+  - You need the .pivotal file and the osdf file at a minimum
+  - For Tanzu Tile software dependencies:
+    - Stemcell: Go to Tanzu Ops Manager, the environment spun up by the self service center. Browse to STEMCELL LIBRARY. If you’ve uploaded your Tanzu Tile you should see which stemcell your Tanzu Tile is dependent upon, and you can use this information to specify to users which stemcell is required.
+    - Ops manager: Go to the self service center. Search in the Credentials: section for ops_manager_version. This can be used to specify which Ops manager version our tile is compatible with.
```

Comment:
```suggestion
    - Ops manager: Go to the self service center. Search in the `Credentials:` section for `ops_manager_version`. This can be used to specify which Ops manager version our tile is compatible with.
```